solutions/quickstart/variables.tf

```diff
@@ -73,3 +73,9 @@ variable "service_credential_names" {
   type        = map(string)
   default     = {}
 }
+
+variable "metrics" {
+  type        = list(string)
+  description = "Enhanced metrics to activate, as list of strings. Allowed values: 'topic', 'partition', 'consumers'."
```

Comment:
The quickstart solution only supports lite and standard plans. Did you not say metrics is only supported for enterprise?

Reply:
I will remove the metrics from quickstart.